src/katsdpmodels/models.py

```diff
@@ -129,6 +133,15 @@ def from_file(cls: Type[_M], file: io.IOBase, url: str, *,
         type; otherwise `url` may be used instead.
         """
 
+    @abstractmethod
+    def to_file(self, file: Union[str, pathlib.Path, _FileLike], *,
```

Comment:
Is there a particular use case for passing in an already parsed pathlib.Path as opposed to just allowing strings and _FileLike ?

Reply:
Just to make it work similarly to the rest of Python e.g. `open` will accept a Path.